Open-Science-Workshop-2022/tutorials/act-basics.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>Can you add plt.show() calls for each of these to make sure the graphics render in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-Development/ARM-Notebooks/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>